apps/svelte.dev/src/routes/banner.json/+server.js

```diff
@@ -9,29 +9,15 @@ export const GET = async () => {
 	return json(
 		defineBanner([
 			{
-				id: 'advent2023',
-				start: new Date('1 Dec, 2023 00:00:00 UTC'),
-				end: new Date('24 Dec, 2023 23:59:59 UTC'),
+				id: 'sveltehack2024',
+				start: new Date('22 Oct, 2024 00:00:00 UTC'),
+				end: new Date('10 January, 2025 23:59:59 UTC'),
```

Comment:
do we really want it to be visible for the entire duration?

Reply:
probably makes sense to put it down mid december, you wouldn't start stuff then.

once you close it won't reappear, if that's your concern of it being annoying.